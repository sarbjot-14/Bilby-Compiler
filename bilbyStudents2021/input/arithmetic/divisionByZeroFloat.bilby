main{

	mut divideerr := 30.0/0.0;

}