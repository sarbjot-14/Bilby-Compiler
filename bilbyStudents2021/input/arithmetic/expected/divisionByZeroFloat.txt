Runtime error: float divide by zero
